astro/src/content/docs/get-started/core-concepts/applications.mdx

```diff
@@ -204,6 +205,15 @@ The multi-factor configuration allows you to provide Application specific multi-
   <APIField name="SMS template" optional>
     When a template is selected, it will be used to send a multi-factor authentication code when the SMS MFA method is used and a user is signing in to this application.
   </APIField>
+  <APIField name="MFA requirement lambda" optional>
+    The MFA requirement lambda that will be invoked during logins, password changes, and MFA Status API calls. FusionAuth uses this lambda to determine whether to challenge the user with MFA. The lambda assigned to the application will override the lambda assigned to the application's tenant.
+
+    You can customize this logic to:
+
+    * Change FusionAuth’s MFA decision for a user.
+    * Trigger a suspicious login event based on your criteria. &nbsp; <AvailableSince since="1.62.0"/>
```

Comment:
Both these items are new in 1.62.0 aren't they? Should the `since` go in the `APIField` attribute?